.vscode/settings.json

```diff
@@ -46,5 +46,6 @@
     "*.astro": "${capture}.tsx, ${capture).css",
     "*.tsx": "${capture}.ts, ${capture}.css",
     "*.stories.tsx": "${capture}.test.ts"
-  }
+  },
+  "cSpell.words": ["dont", "itwin", "itwinui", "Tbgr"]
```

Comment:
undo this change. If you want to add some terms to your dictionary, you can always do it for your "User settings" rather than "Workspace settings" in the dropdown menu